Dockerfile

```
@@ -0,0 +1,31 @@
+FROM python:3-alpine AS base
+
+WORKDIR /opt/app
+
+# Install Poetry
+RUN pip install pipx
+RUN pipx install poetry
+RUN ln -s ~/.local/bin/poetry /usr/bin/poetry
```

Definitely falling somewhat into premature-optimisation here, but worth noting that one of the first pieces of advice if we are looking to reduce image sizes is to combine layers, and these three might be a nice example of where that feels reasonable:
```suggestion
RUN pip install pipx && pipx install poetry && ln -s ~/.local/bin/poetry /usr/bin/poetry
```